### PR TITLE
[Tools] Drop compare_bench.py, compare.py is to be used, add U-test docs

### PR DESCRIPTION
**Pull request by LebedevRI**
As discussed in IRC, time to deduplicate.

### CODE REVIEW

File: docs/tools.md [hunk 1]
```diff
@@ -1,83 +1,13 @@
...
-As you can note, the values in `Time` and `CPU` columns are calculated as `(new - old) / |old|`.
-Obviously this example doesn't give any useful output, but it's intended to show the output format when 'compare_bench.py' needs to run benchmarks.
+### Modes of operation
 
-## compare.py
-
-The `compare.py` can be used to compare the result of benchmarks.
 There are three modes of operation:
 
 1. Just compare two benchmarks, what `compare_bench.py` did.
```
Comment (dominichamon) [inline docs/tools.md]: drop this line about compare_bench?

**Comment by LebedevRI** [comment]
Ping?

**Comment by dominichamon** [comment]
Sorry, I thought you were going to accept and use committer rights ;)

**Comment by LebedevRI** [comment]
> use committer rights ;)

Uhm, i'm not sure this is how it works :)
It [normally] never is a pass to do whatever one wants.

**Pull request merged**